.github/workflows/main.yml

```
@@ -365,6 +374,8 @@ jobs:
           test -f h2o2-test.yaml
           cti2yaml test/data/ch4_ion.cti ch4_ion-test.yaml
           test -f ch4_ion-test.yaml
+          yaml2ck --input=data/h2o2.yaml --mechanism=h2o2-test.ck
```

```suggestion
          yaml2ck data/h2o2.yaml --mechanism=h2o2-test.ck
```
Same for other instance